### PR TITLE
/kg -> /browser

### PR DESCRIPTION
**Pull request by beets**
* Move kg routes into a module
* Convert dcid get param to path args
* Add tests, update links sitewide
* Add back dc.org homepage
* Remove unused template

### CODE REVIEW

File: server/templates/browser/kg_main.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    <p>
+      The Data Commons Graph (DCG) is constructed by synthesizing a single Graph from many different data sources.
+      It currently contains data from Wikipedia, the US Census, NOAA, FBI, etc.
+      The browser can be used both to explore what data is available and to understand the graph structure, to help use the API.
+    </p>
+    <p>
+      You can use the search box above or start looking around from some well known starting points:
+    </p>
+    <ul>
+      <li class="type-list">
```
Comment (shifucun) [inline server/templates/browser/kg_main.html:35]: Add a todo to pass a json from the route to here and use for loop for these lists?

File: server/templates/homepage.html [hunk 2]
```diff
@@ -12,112 +12,68 @@
...
+      contribute.
+    </p>
 
-{% block content %}
-  <div id="homepage-content">
     <p>
-      The Data Commons Graph (DCG) is constructed by synthesizing a single Graph from many different data sources.
-      It currently contains data from Wikipedia, the US Census, NOAA, FBI, etc.
-      The browser can be used both to explore what data is available and to understand the graph structure, to help use the API.
+      You can use the <a href="https://browser.datacommons.org">Data Commons
```
Comment (shifucun) [inline server/templates/homepage.html:62]: update this?

File: server/templates/homepage.html [hunk 3]
```diff
@@ -12,112 +12,68 @@
...
+      Graph Browser</a> to browse the graph. The data can be
+      programmatically accessed via <a href="http://docs.datacommons.org/api">APIs</a>.
```
Comment (shifucun) [inline server/templates/homepage.html:64]: will this link change too?

**Comment by shifucun** [comment]
can comment "@googlebot I signed it!" so the cla check can pass

File: server/templates/browser/kg_main.html (hunk 1 above)
Comment (beets) [reply server/templates/browser/kg_main.html:35]: Done. Will probably keep the config in python

File: server/templates/homepage.html (hunk 2 above)
Comment (beets) [reply server/templates/homepage.html:62]: Will be deleted in the next PR

File: server/templates/homepage.html (hunk 3 above)
Comment (beets) [reply server/templates/homepage.html:64]: Not for now...

**Comment by beets** [comment]
Made the suggested updates. PTAL!

**Pull request merged**